var/spack/repos/builtin/packages/openexr/package.py

```diff
@@ -6,14 +6,14 @@
 from spack import *
 
 
-class Openexr(AutotoolsPackage):
+class Openexr(CMakePackage):
     """OpenEXR Graphics Tools (high dynamic-range image file format)"""
 
     homepage = "http://www.openexr.com/"
-    url = "https://github.com/openexr/openexr/releases/download/v2.3.0/openexr-2.3.0.tar.gz"
+    url      = "https://github.com/AcademySoftwareFoundation/openexr/archive/v2.5.3.tar.gz"
 
-    # New versions should come from github now
-    version('2.3.0', sha256='fd6cb3a87f8c1a233be17b94c74799e6241d50fc5efd4df75c7a4b9cf4e25ea6')
+    # New Version Should Come From GitHub
+    version('2.5.3', sha256='6a6525e6e3907715c6a55887716d7e42d09b54d2457323fcee35a0376960bebf')
```

Comment:
unless there's a good reason to remove 2.3.0, 2.5.3 should be added not replace it